docs/modules/operation/pages/quick-start/users.adoc

```diff
@@ -48,12 +46,13 @@ You should still change the password of the `admin` user to be more secure.
 [[ga-users-next]]
 == Next steps
 
-Before xref:operation:quick-start/notifications.adoc[setting up default notifications], you must configure notification information for the `admin` user profile:
+Before xref:operation:quick-start/notifications.adoc[setting up default notifications], you must configure notification information for any users with admin privileges:
```

Comment:
Because this is a quick start, so far they have only one user, the Admin user, so there aren't any other users to modify. While we referred them to the create other users section, we really only want them to create one user at this point. 
Incorporated this content above or we can update the current procedure below to be "Create a new user".